test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -885,6 +885,20 @@ public void GenerateSchema_HonorsSerializerAttribute_JsonPropertyName()
             Assert.Equal(new[] { "string-with-json-property-name" }, schema.Properties.Keys.ToArray());
         }
 
+#if NET7_0_OR_GREATER
+        [Fact]
+        public void GenerateSchema_HonorsSerializerAttribute_JsonRequired()
+        {
+            var schemaRepository = new SchemaRepository();
+
+            var referenceSchema = Subject().GenerateSchema(typeof(JsonRequiredAnnotatedType), schemaRepository);
+
+            var schema = schemaRepository.Schemas[referenceSchema.Reference.Id];
+            Assert.Equal(new[] { "StringWithJsonRequired" }, schema.Required.ToArray());
```

Comment:
Can this be simplified with collection expressions?

Reply:
Call is ambiguous unless ToArray.ToSpan(), as the rest of the methods are in the same way I keep this way